src/components/AutoCompleteSuggestions/index.tsx

```diff
@@ -126,22 +145,24 @@ function AutoCompleteSuggestions<TSuggestion>({measureParentContainerAndReportCu
             } else {
                 // calculation for big suggestion box below the cursor
                 measuredHeight = measureHeightOfSuggestionRows(suggestionsLength, true);
-                bottomValue = windowHeight - y - cursorCoordinates.y + scrollValue - measuredHeight - CONST.AUTO_COMPLETE_SUGGESTER.SUGGESTION_ROW_HEIGHT - keyboardHeight;
+                bottomValue = windowHeight - y - effectiveCursorY + scrollValue - measuredHeight - CONST.AUTO_COMPLETE_SUGGESTER.SUGGESTION_ROW_HEIGHT - keyboardHeight;
             }
 
             setSuggestionHeight(measuredHeight);
             setContainerState({
                 left: leftValue.current,
                 bottom: bottomValue,
                 width: widthValue,
-                cursorCoordinates,
+                cursorCoordinates: {x: effectiveCursorX, y: effectiveCursorY},
             });
         });
     }, [measureParentContainerAndReportCursor, windowHeight, windowWidth, keyboardHeight, shouldUseNarrowLayout, suggestionsLength, bottomInset, topInset, isKeyboardAnimatingRef]);
 
-    if ((containerState.width === 0 && containerState.left === 0 && containerState.bottom === 0) || (containerState.cursorCoordinates.x === 0 && containerState.cursorCoordinates.y === 0)) {
+    // Only prevent rendering if we have no suggestions
+    if (!suggestionsLength) {
```

Comment:
If that, I think we only need to replace the second condition

```suggestion
    if ((containerState.width === 0 && containerState.left === 0 && containerState.bottom === 0) || !suggestionsLength) {
```

Reply:
Given that the validation check for the container coordinates could still be useful to prevent things like flickering/unnecessary rendering, I can add the check back. The cursor coordinates were what were causing issues originally. Should I commit your suggestion or do you want me to add this on my end and push a separate commit?

Reply:
> Should I commit your suggestion or do you want me to add this on my end and push a separate commit?

It's up to you. Both ways are fine

Reply:
@danstepanov And please also remove the above comment

```
    // Only prevent rendering if we have no suggestions
```

Reply:
Yeah, but it will be outdated now. Or please update this to reflect the new change